### PR TITLE
@ #492 | fix msg is not string bug

### PR DESCRIPTION
**Pull request by datphan**
connect #492

### CODE REVIEW

**Review by hieptranquoc — DISMISSED** [review]

**Comment by hoatle** [comment]
https://github.com/teracyhq/dev/blob/develop/lib/teracy-dev/logging.rb#L62 is the line should be the place for bug fix, not on the mask_filter file.

**Review by hoatle — CHANGES_REQUESTED** [review]
not the right place for fix

File: lib/teracy-dev/logging.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@ def self.configure_logger_for(classname)
 
         if accepted
           @@filters.each do |filter|
-            msg = filter.filtered(msg)
+            msg = filter.filtered(msg.to_s)
```
Comment (datphan) [inline lib/teracy-dev/logging.rb:62]: @hoale updated, pls check again

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**